bin/wasabi.sh

```diff
@@ -213,12 +222,14 @@ package() {
     done; \
     sed -i '' -e "s|http://localhost:8080|${server}|g" target/constants.json 2>/dev/null; \
     sed -i '' -e "s|VERSIONLOC|${version}|g" target/app/index.html 2>/dev/null; \
+    if [ -z "$OS" ]; then \
     (cd target; \
       npm install; \
       bower install; \
       grunt clean; \
       grunt build --target=develop --no-color); \
```

Comment:
Sorry, @jwtodd , but I'm not sure what the context is.  Are you asking what we should do related to the UI when doing a jenkins build internally?  I guess it depends on whether you want to build and install the UI?  If so, then you need those commands.  It seems that "-z "$OS"" is a check if that var is empty, right?  I don't see where it gets set in the script.  But basically, those commands are necessary if you want the UI built at all.

Also, you mention something about grunt being pre-installed.  I don't think it is pre-installed on any system, certainly, not on our laptops.  It is on jenkins, because it was installed globally by hand (by me, probably).  So, again, not sure what you're asking.